esx_service/vmdk_ops_test.py

```diff
@@ -193,6 +194,20 @@ def testAttachAsOpts(self):
         err = vmdk_ops.removeVMDK(self.name)
         self.assertEqual(err, None, err)
 
+    def test_VerifyVolumeSize(self):
+          """ test to verify the size of vmdk is same by
+	      checking the capacity of vmdk using vmkfstools """
+	err = vmdk_ops.createVMDK(vm_name=self.vm_name,
+	                          vmdk_path=self.name,
+				  vol_name=self.volName)
+	self.assertEqual(err, None, err)
+	self.assertEqual(
+	     os.path.isfile(self.name), True,
+	     "VMDK {0} is missing after create.".format(self.name))
+	response = vmdkops_admin.get_vmdk_size_info(path=self.name)
+	self.assertEqual(response["capacity"], "100.00MB",
+	                 "Expected size of volume = 100 MB. Actual size = {0}.
+			 Incorrect size for VMDK {1}".format(response["capacity"], self.name))
```

Comment:
That's true! 

My opinion is to call vmfs tool directly instead of calling "vmdkops_admin.get_vmdk_size_info()" and compare the bytes returned from vmkfs tool with the expected one instead of going through the route of  get_vmdk_size_info & human_readable (which is consumed by 'vmdkops_admin.py ls')